azure-pipelines.yml

```diff
@@ -201,6 +201,17 @@ jobs:
       buildType: 'current'
       downloadType: 'specific'
       downloadPath: '$(Build.SourcesDirectory)'
+  - task: ManifestGeneratorTask@0
+    displayName: 'SBOM Generation Task'
+    inputs:
+      BuildDropPath: '$(Build.SourcesDirectory)'
+      BuildComponentPath: '$(Build.SourcesDirectory)'
+      Verbosity: 'Verbose'
+  - task: CopyFiles@2
+    inputs:
+      SourceFolder: '$(Build.SourcesDirectory)\_manifest'
+      Contents: '**'
```

Comment:
Shouldn't contents be '*.*'?

Reply:
as per doc it says the following .. so I chose default case to make sure I'm not missing any subfolder contents
![image](https://user-images.githubusercontent.com/16231002/144118319-8c95f2e0-c3f2-4957-9a91-da8d635c371c.png)